test-requirements.txt

```diff
@@ -5,6 +5,7 @@ pip>=9.0
 -r requirements.txt
 
 # Test runner:
+codecov
```

Comment:
It's not about runner, please put it into a separate section, like `# Reporting:`